### DISCUSSION TITLE
Connecting to database

### DISCUSSION THREAD

**Discussion by jdgunaseelan** [POPWorldMedia/POPForums#242] [Q&A]
I am testing my integration of PopForums v13.0.2, but when I do I get a nullreference error in regards to the Isqlobjectfactory. I'm assuming this has to do with the Connection string. Would I just add the connection string of my database into the config files, and would it need to be done to all the config files that use connection string?

**Comment by jeffputz** [comment]
I have no idea. What's the stack trace? What happens when you put a debugger on it?

**Reply by jdgunaseelan** [reply]
This is what I get when debugging. Object reference not set to an instance of an object.
![PopF](https://user-images.githubusercontent.com/91080783/135278865-1bd3ae23-f3df-4037-83f2-a38f87e231a6.PNG)

**Comment by jeffputz** [comment]
I imagine that your dependency injection isn't happening.

**Reply by jdgunaseelan** [reply]
So would I have to mess with the connection strings? I haven't really touched the PopForums code, I've just called some of the methods.

**Reply by jeffputz** [reply]
No, you need to set up the DI container. There's no connection factory because one isn't being inserted when an instance of the class is being created. DI is far beyond the scope of what I can support.
